tests/FullApplicationTest.php

```diff
@@ -610,6 +610,15 @@ public function testRequestUser()
         $this->assertSame('1234', $response->getContent());
     }
 
+    public function testCanResolveFilesystemFactoryFromContract()
+    {
+        $app = new Application();
+
+        $filesystem = $app['Illuminate\Contracts\Filesystem\Factory'];
+
+        $this->assertInstanceOf('Illuminate\Contracts\Filesystem\Factory', $filesystem);
```

Comment:
Please use the `::class` notation for this and the above reference.

Reply:
@driesvints most of the tests here don't use the `::class` notation. I've updated this one but it might be worth updating the rest of the tests too.

Reply:
@Kieran-Brown yeah, I'm aware. Just want to prevent any more from getting in. Thanks!